package-lock.json

```diff
@@ -0,0 +1,4337 @@
+{
```

Comment:
Remove

Reply:
@sushantdhiman as in remove the package-lock.json file?